bin/all_sky_search/pycbc_dtphase

```diff
@@ -1,36 +1,39 @@
 #!/bin/env python
-""" Create a file containing the time, phase and amplitude
-correlations between two or more detectors for signals by
-doing a simple monte-carlo
+
+"""
+Create a file containing the time, phase and amplitude correlations between two
+or more detectors for signals by doing a simple monte-carlo.
 
 Output is the relative amplitude, time, and phase as compared to a reference
-IFO. A separate calculation is done for each IFO as a possible reference.
-The data is stored as two vectors : one vector gives the discrete integer bin
+IFO. A separate calculation is done for each IFO as a possible reference.  The
+data is stored as two vectors : one vector gives the discrete integer bin
 corresponding to a particular location in 3*(Nifo-1)-dimensional
 amplitude/time/phase space, the other gives the weight assigned to that bin.
 To get the signal rate this weight should be scaled by the local sensitivity
 value and by the SNR of the event in the reference detector.
 """
-import argparse, h5py, numpy as np, pycbc.detector, logging, multiprocessing
-from numpy.random import normal, uniform, power
+
+import argparse, h5py, numpy as np, pycbc.detector, logging
+from numpy.random import uniform
 from scipy.stats import norm
 from copy import deepcopy
 
-parser = argparse.ArgumentParser()
+
+parser = argparse.ArgumentParser(description=__doc__)
```

Comment:
Well that's a useful thing I never knew about!

Reply:
I do love that!